src/cong.h

```diff
@@ -395,50 +393,50 @@ namespace libsemigroups {
     //!
     //! \warning Any existing data for the congruence is deleted by this
     //! method, and may have to be recomputed. The return values and runtimes
-    //! of other methods applied to \c this may also be effected.
+    //! of other methods applied to \c this may also be affected.
     //!
     //! \warning The Todd-Coxeter Algorithm may never terminate when applied to
     //! a finitely presented semigroup.
     void force_tc_prefill();
 
-    //!  Use an elementary orbit algorithm which enumerates pairs of
-    //! Element's that are related in \c this.
+    //! Use an elementary orbit algorithm which enumerates pairs of
+    //! Element objects that are related in \c this.
     //!
-    //! The method that this method forces the Congruence to use, is unlikely
+    //! The method forced by this is unlikely
     //! to terminate, or to be faster than the other methods, unless there are
     //! a relatively few pairs of elements related by the congruence.
     //!
+    //! This method only applies to a congruence created using a Semigroup
+    //! object, and does not apply to finitely presented semigroups.
+    //!
     //! \warning Any existing data for the congruence is deleted by this
     //! method, and may have to be recomputed. The return values and runtimes
-    //! of other methods applied to \c this may also be effected.
+    //! of other methods applied to \c this may also be affected.
     //!
-    //! \warning The algorithms that this method forces a Congruence to use
-    //! will only terminate if there are finitely many pairs of elements
-    //! related by \c this. The worst case space complexity of these algorithms
+    //! \warning The worst case space complexity of these algorithms
     //! is the square of the size of the semigroup over which \c this is
     //! defined.
     void force_p();
 
-    //!  Use the Knuth-Bendix algorithm on a rewriting system RWS with
-    //! rules
+    //! Use the Knuth-Bendix algorithm on a rewriting system RWS with rules
     //! obtained from Congruence::relations followed by an elementary orbit on
     //! pairs method on the resulting semigroup.
     //!
     //! This method forces the use of the [Knuth-Bendix
     //! algorithm](https://en.wikipedia.org/wiki/Knuth–Bendix_completion_algorithm)
     //! to compute the congruence defined by the generators and relations of \c
     //! this followed by an elementary orbit algorithm which enumerates pairs
-    //! of Element's of that semigroup which are related in \c this.
+    //! of Element objects of that semigroup which are related in \c this.
     //! Knuth-Bendix is applied to a rewriting system obtained from
     //! Congruence::relations.
     //!
-    //! At present the Knuth-Bendix algorithm can only be applied to two-sided
-    //! congruences, and this method asserts that \c this is a two-sided
-    //! congruence.
+    //! This method only applies to a congruence over a finitely presented
```

Comment:
As we discussed yesterday, the KBP method does actually apply to left and right congruences, since it only runs Knuth-Bendix on `_relations` (i.e. the two-sided ones).